src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -107,11 +107,18 @@ function ChartDeploymentHistory({
             })
     }, [])
 
+    const getDeploymentData = (_selectedDeploymentTabIndex: number, _selectedDeploymentHistoryIndex: number) => {
+        if (_selectedDeploymentTabIndex === 1 || _selectedDeploymentTabIndex === 2) {
+            const version = deploymentHistoryArr[_selectedDeploymentHistoryIndex].version
+            checkAndFetchDeploymentDetail(version)
+        }
+    }
+
     function changeDeployment(index: number) {
```

Comment:
what is the difference between changeDeployment and changeDeploymentTab

Reply:
One is for tabs and another is for the sidebar. Rnamed more specific now